lib/3scale_toolbox/commands/find_command/find_account.rb

```diff
@@ -0,0 +1,39 @@
+require 'cri'
+require '3scale_toolbox/base_command'
+require '3scale_toolbox/entities/account'
+
+module ThreeScaleToolbox
+  module Commands
+    module FindCommand
+      class FindAccountSubcommand < Cri::CommandRunner
+        include ThreeScaleToolbox::Command
+
+        def self.command
+          Cri::Command.define do
+            name        'account'
+            usage       'account <text>'
+            summary     'find account'
+            description 'Find account by email, provider key or service token'
+
+            option :i, :instance, '3scale instance. Url or remote name', argument: :required
+            option :a, :'print-all', 'Print all the account info', argument: :forbidden
+            param :text
+
+            runner FindAccountSubcommand
+          end
+        end
+
+        def run
+          client = threescale_client(options[:instance])
```

Comment:
instance is an optional param. What if it is not given in the command? I would change it to positional param and therefore enforced by the CRI lib to exist